models/request_result.rb

```diff
@@ -1,3 +1,5 @@
+require 'securerandom'
```

Comment:
I'm a little confused about this. Was `send_message` not working before and if it was, why are we now requiring `securerandom`?

Reply:
It's a good question, and honestly I do not know why SecureRandom presumably does *not* need to be explicitly required in deployed code. It does locally. Here's what I get in a local 2.2.2 irb:

```
2.2.2 :001 > SecureRandom
NameError: uninitialized constant SecureRandom
	from (irb):1
	from /Users/paulbeaudoin/.rvm/rubies/ruby-2.2.2/bin/irb:11:in `<main>'
2.2.2 :002 > require 'securerandom'
 => true 
2.2.2 :003 > SecureRandom
 => SecureRandom 
```

I assume SecureRandom is automatically required in the traveling ruby setup, but at a glance I can't say how. I assume running `require 'securerandom'` when it has already been required will be a noop once deployed.